### PR TITLE
Support old variables

### PR DESCRIPTION
**Pull request by AlexProgrammerDE**
Change-type: patch

### CODE REVIEW

**Comment by MatthewCroughan** [comment]
What I'm more annoyed about is that the commit no longer includes my actual changes, but includes changes from other people. I'm really confused about this. It's interesting to see CI cause so much trouble for such a simple change.

File: airplay/start.sh [hunk 1]
```diff
@@ -10,17 +10,20 @@ if [[ -z $DISABLE_MULTI_ROOM ]] && [[ $CLIENT_ONLY_MULTI_ROOM == "1" ]]; then
   exit 0
 fi
 
-
 # Set the device broadcast name for AirPlay
 if [[ -z "$DEVICE_NAME" ]]; then
   DEVICE_NAME=$(printf "balenaSound Airplay %s" $(hostname | cut -c -4))
 fi
 
+if [[ "$BLUETOOTH_DEVICE_NAME" ]]; then
```
Comment (tmigone) [inline airplay/start.sh:18]: That was fast, awesome work! I'm super nitpicking here... but if a user has both variables set, then `BLUETOOTH_DEVICE_NAME` will end up being used. I think it's better if we stick with `DEVICE_NAME` in that case. Something like:

```
if [[ -z "$DEVICE_NAME" ]]; then
   if [[ "$BLUETOOTH_DEVICE_NAME" ]]; then
     DEVICE_NAME="$BLUETOOTH_DEVICE_NAME"
   else
     DEVICE_NAME=$(printf "balenaSound Airplay %s" $(hostname | cut -c -4))
   fi
fi
```

**Comment by chrisys** [comment]
@tmigone I agree, and also wonder if we *need* to provide the fallback for Airplay/Spotify, as it could be argued it was incorrect to use a "Bluetooth device name" for those services anyway.

**Comment by tmigone** [comment]
Yeah I think Chris is right, that was incorrect from the get go. I vote we only keep this fix in the bluetooth container then.

**Comment by AlexProgrammerDE** [comment]
@balena-ci retest

**Comment by AlexProgrammerDE** [comment]
@balena-ci retest

**Comment by AlexProgrammerDE** [comment]
@tmigone is this version for you ok?

**Comment by tmigone** [comment]
lgtm!

**Pull request merged**